code/modules/mob/living/living.dm

```diff
@@ -1575,6 +1575,14 @@
 		return BODYTEMP_NORMAL
 	return BODYTEMP_NORMAL + get_body_temp_normal_change()
 
+///Returns the body temperature at which this mob will start taking heat damage. Gets overridden in carbon/human/life.dm. Not actually intended for usage by non-carbons; this proc being defined at the living level is just to match get_body_temp_normal() (and for safety's sake).
+/mob/living/proc/get_body_temp_heat_damage_limit()
```

Comment:
isnt what this stuff will return fake for simplemobs or like aliens?

Reply:
the comments of this proc explicitly mention that it should only be used with carbons

I'll move it down to the carbon level, at least

Reply:
actually, hm. get_body_temp_normal() is up here at the living level, so maybe this should remain here as well